src/Cli/dotnet/Installer/Windows/Security/UnionChoice.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Installer.Windows.Security
+{
+    /// <summary>
+    /// Specifies the union member to use when verifying trust.
+    /// </summary>
+    public enum UnionChoice : uint
+    {
+        /// <summary>
+        /// Use the file pointed to by pFile. See <see cref=""/>
```

Comment:
`See <see cref=""/>`? Should there be a link there?

Reply:
I'll fix that